pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -1010,6 +1013,14 @@ func pvcBackedOSDEnvVar(pvcBacked string) v1.EnvVar {
 	return v1.EnvVar{Name: pvcBackedOSDVarName, Value: pvcBacked}
 }
 
+func logLevelEnvVar(debug bool) v1.EnvVar {
```

Comment:
```suggestion
func setDebugLogLevelEnvVar(debug bool) v1.EnvVar {
```

So that the bool is more meaningful?